docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -1842,6 +1848,31 @@ Comma-separated list of attributes to include in all new spans, such as `key1:va
 
 Can be set with the environment variable `OTEL_RESOURCE_ATTRIBUTES` (use `=` instead of `:` with the environment variable).
 
+### sampler_type
+
+Default value is `const`.
+
+Specifies the type of sampler: `const`, `probabilistic`, `ratelimiting`, or `remote`.
+
+### sampler_param
+
+Default value is `1`.
+
+Sampler configuration parameter. Depending on the value of `sampler_type`, it can be `0`, `1`, or a decimal value in between.
+
+- For `const` sampler, `0` or `1` to always `false`/`true` respectively
+- For `probabilistic` sampler, a probability between `0.0` and `1.0`
+- For `rateLimiting` sampler, the number of spans per second
+- For `remote` sampler, param is the same as for `probabilistic`
+  and indicates the initial sampling rate before the actual one
+  is received from the sampling server
+
+### sampling_server_url
+
+When `sampler_type` is `remote`, this specifies the URL of the sampling server. This can be used by all tracing providers. 
+
+The sampling server should implement the Jaeger remote sampling API, which can be exposed by a jaeger-agent, jaeger-collector, opentelemetry-collector-contrib, or the Grafana Agent.
```

Comment:
does the user do this implementation? I'm wondering if this should be something like "In the sampling server, implement the Jaeger remote sampling API...".

Reply:
No, the user just needs to choose a server that implements that API - the 4 listed all implement the Jaeger remote sampling API.

Reply:
👍 , all good on this one then.

Reply:
How about:

```suggestion
Use a sampling server that implement the Jaeger remote sampling API, such as jaeger-agent, jaeger-collector, opentelemetry-collector-contrib, or Grafana Agent.
```

(maybe `Grafana Agent` should link to https://grafana.com/oss/agent/?)

Reply:
@chri2547 I think I was busy writing this comment and didn't see your original reply - what do you think of the alternate phrasing?

Reply:
I like that, tho would use "to" instead of "that" - so "Use a sampling server to implement..."

Reply:
oops - it should've been `that implements`

"Use a sampling server to implement ..." doesn't really scan well to me - the user's really just running some server and putting its URL in this setting. The important thing is the server needs to support the correct API. Actually, maybe "supports" is a clearer term. Or maybe I'm bikeshedding myself at this point 😂

Reply:
How about:

```suggestion
Use a sampling server that supports the Jaeger remote sampling API, such as jaeger-agent, jaeger-collector, opentelemetry-collector-contrib, or [Grafana Agent](https://grafana.com/oss/agent/).
```

Reply:
👍